### PR TITLE
Added Histogram X-Axis Range Change tutorial

### PR DESCRIPTION
**Pull request by AdvaitDhingra**
# This Pull request:

## Changes or fixes:

Adds a tutorial demonstrating how the changing of the range can zoom into the histogram.

As discussed in [this](https://root-forum.cern.ch/t/histogram-x-axis-range/12815) forum post.

## Checklist:

- [x] tested changes locally
- [x] updated the docs (if necessary)

### CODE REVIEW

File: tutorials/hist/RangeChange.C [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+///
+/// This demonstrates how to zoom into a Histogram by
+/// changing the range on one of the axes (or both).
+///
+/// \macro_image
+/// \macro_code
+///
+/// \author Advait Dhingra
+
+void RangeChange() {
```
Comment (Axel-Naumann) [inline tutorials/hist/RangeChange.C:16]: `ZoomHistogram.C`?

**Comment by AdvaitDhingra** [comment]
That should do it

**Comment by AdvaitDhingra** [comment]
@Axel-Naumann  does everything look good to you?

File: tutorials/hist/ZoomHistogram.C [hunk 2]
```diff
@@ -0,0 +1,38 @@
+/// \file
+/// \ingroup tutorial_hist
+/// \notebook -js
+/// Changing the Range on the X-Axis of a Histogram
+///
+/// Image produced by `.x ZoomHistogram.C`
+///
+/// This demonstrates how to zoom into a Histogram by
```
Comment (Axel-Naumann) [inline tutorials/hist/ZoomHistogram.C:8]:
```suggestion
/// This demonstrates how to zoom into a histogram by
```

File: tutorials/hist/ZoomHistogram.C [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+/// \author Advait Dhingra
+
+void ZoomHistogram() {
+
+   TH1F *norm = new TH1F("Normal Histogram", "Normal Histogram", 100, 0, 100);
+
+   TRandom2 rand;
+
+   for (int i = 0; i < 100; ++i) {
+     Double_t j = rand.Gaus(50, 10);
```
Comment (Axel-Naumann) [inline tutorials/hist/ZoomHistogram.C:23]:
```suggestion
   for (int i = 0; i < 100; ++i) {
     Double_t j = gRandom->Gaus(50, 10);
```

File: tutorials/hist/ZoomHistogram.C [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+     norm->Fill(j);
```
Comment (Axel-Naumann) [inline tutorials/hist/ZoomHistogram.C:24]:
```suggestion
     Double_t x = rand.Gaus(50, 10);
     norm->Fill(x);
```
"j" sounds like an inner index (compared to "i"), where really it's a value on the x-axis.

**Review by Axel-Naumann — CHANGES_REQUESTED** [review]
Tiny improvements still - thanks! Ping me when you're done so I can merge

File: tutorials/hist/ZoomHistogram.C (hunk 4 above)
Comment (AdvaitDhingra) [reply tutorials/hist/ZoomHistogram.C]: Good point

**Comment by AdvaitDhingra** [comment]
@phsft-bot abort

**Comment by Axel-Naumann** [comment]
I've stopped the bot :-)

**Review by Axel-Naumann — APPROVED** [review]

**Pull request merged**